doc/sphinx/source/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# TDycore Code of Conduct
```

Comment:
It looks like you've made a copy here, but we used symlinks in libCEED so the same file isn't duplicated.